src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarExtraLinks.tsx

```diff
@@ -27,39 +29,88 @@ const Link = styled.a`
   text-decoration: underline;
 `
 
+@track()
 class ArtworkSidebarExtraLinksContainer extends React.Component<
   ArtworkSidebarExtraLinksContainerProps
 > {
+  @track(() => ({
+    context_module: Schema.ContextModule.Sidebar,
```

Comment:
You can set the `context_module` in the top-level `@track` on 32 and it will pass down to all `@track` calls in the component.